src/app.js

```diff
@@ -1 +1,20 @@
+/* eslint-disable no-console */
 'use strict';
+
+const terminal = (process.argv).slice(2);
```

Comment:
you can use destruction to get operation (add, multiply) and array with elements (use rest)